### PR TITLE
Add issue reporting for stress builds with timeout and panics.

### PR DESCRIPTION
**Pull request by BramGruneir**
This looks for the last test in the run that failed due to a panic and files an issue for it.

### CODE REVIEW

File: scripts/run_stress.sh [hunk 1]
```diff
@@ -101,7 +101,7 @@ function json_escape(){
 }
 
 function post() {
-  curl -X POST -H "Authorization: token ${GITHUB_API_TOKEN}" "https://api.github.com/repos/cockroachDB/cockroach/issues" -d "${1:0:30000}"
+  curl -X POST -H "Authorization: token ${GITHUB_API_TOKEN}" "https://api.github.com/repos/cockroachDB/cockroach/issues" -d "${1}"
```
Comment (bdarnell) [inline scripts/run_stress.sh]: Why did you remove this length limit? We've had problems before with stack traces being too big to post via curl (using `curl -d`. you would still be able to write this data to a file and post it that way).

Comment (BramGruneir) [inline scripts/run_stress.sh]: done

**Comment by mberhault** [comment]
LGTM. I ran the new chunk, and it seems ok. I presume you've done a full run.
